### PR TITLE
Syntax to using where

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

**Review by Sacha0 — APPROVED** [review]
Thanks @musm!

File: base/range.jl [hunk 1]
```diff
@@ -859,7 +859,7 @@ function in(x, r::Range)
 end
 
 in(x::Integer, r::AbstractUnitRange{<:Integer}) = (first(r) <= x) & (x <= last(r))
-in{T<:Integer}(x, r::Range{T}) = isinteger(x) && !isempty(r) && x>=minimum(r) && x<=maximum(r) && (mod(convert(T,x),step(r))-mod(first(r),step(r)) == 0)
+in(x, r::Range{T}) where {T<:Integer} = isinteger(x) && !isempty(r) && x>=minimum(r) && x<=maximum(r) && (mod(convert(T,x),step(r))-mod(first(r),step(r)) == 0)
```
Comment (tkelman) [inline base/range.jl]: line length (also for colon and range)

File: base/markdown/parse/parse.jl [hunk 2]
```diff
@@ -73,7 +73,7 @@ parseinline(s::AbstractString, md::MD, c::Config) =
     parseinline(IOBuffer(s), md, c)
 
 # TODO remove once GH #9888 is fixed
```
Comment (tkelman) [inline base/markdown/parse/parse.jl]: that was closed quite a while ago

Comment (musm) [inline base/markdown/parse/parse.jl]: opening a seperate PR for this just in case

**Review by Sacha0 — APPROVED** [review]
lgtm for the two additional commits! (Absent objections, requests for time, or someone else beating me to it, I plan to merge this tomorrow morning. Best!)

**Pull request merged**